ide/static/js/content.js

```diff
@@ -242,8 +260,8 @@ class Content extends React.Component {
     Object.keys(positions).forEach(layerId => {
       const layer = net[layerId];
       layer.state = {
-        top: `${250 + 50 * positions[layerId][1]}px`,
-        left: `${20 + 180 * positions[layerId][0]}px`,
+        top: `${height + 65 * positions[layerId][1]}px`,
+        left: `${width + 80 * positions[layerId][0]}px`,
```

Comment:
I'm not sure I still follow. Why is the offset hardcoded? Still seems a little off.
![screen shot 2017-05-13 at 9 32 56 am 3](https://cloud.githubusercontent.com/assets/8670301/26025891/3f383366-37bf-11e7-8ffe-9df30dd683f4.png)